aggregator/entry.go

```diff
@@ -28,60 +28,76 @@ import (
 	"sync/atomic"
 	"time"
 
+	"github.com/m3db/m3aggregator/bitset"
 	"github.com/m3db/m3aggregator/rate"
 	"github.com/m3db/m3aggregator/runtime"
 	"github.com/m3db/m3metrics/aggregation"
+	"github.com/m3db/m3metrics/metadata"
 	metricid "github.com/m3db/m3metrics/metric/id"
 	"github.com/m3db/m3metrics/metric/unaggregated"
+	"github.com/m3db/m3metrics/op/applied"
 	"github.com/m3db/m3metrics/policy"
 	xerrors "github.com/m3db/m3x/errors"
 
 	"github.com/uber-go/tally"
 )
 
+const (
+	initialAggregationCapacity = 2
```

Comment:
why use 2 here?

Reply:
Mainly just because most metrics will have `10s:2d` and `1m:40d` by default, so the initial capacity of 2.

Reply:
Sure thing.